housekeeping/src/test/java/com/hotels/housekeeping/service/impl/FileSystemHousekeepingServiceTest.java

```diff
@@ -328,7 +328,7 @@ public void eventuallyConsistentCleanUpRemainingPartition() throws Exception {
     // return empty list as val3Parh is still in use.
     when(legacyReplicationPathRepository
         .findByCreationTimestampLessThanEqual(eq(now.getMillis()), any(PageRequest.class)))
-            .thenReturn(new PageImpl<>(Collections.<LegacyReplicaPath>emptyList()));
+            .thenReturn(new PageImpl<>(Collections.<LegacyReplicaPath> emptyList()));
```

Comment:
Ah, I see the remains of an old eclipse formatter. Could you please update the formatter and save the file again so this commit doesn't add a checkstyle warning? 🙂

Reply:
ah yes nice one, I haven't updated thanks! will do.